### DISCUSSION TITLE
how to fix this error

### DISCUSSION THREAD

**Discussion by SaeedAhmed99** [wintercms/winter#838] [Q&A] [answered]
This error appears when downloading plugins
![‏‏لقطة الشاشة (8)](https://user-images.githubusercontent.com/87546758/218677916-58a41d73-4584-4610-a3a0-bc04a6fa469f.png)

**Comment by bennothommo** [comment, marked as answer]
@SaeedAhmed99 the Plugins & Updates search won't fully work until we have our marketplace online. For now, you need to use Composer to install plugins and themes.

**Reply by SaeedAhmed99** [reply]
@bennothommo Please send me documents showing the downloading of plugins from the composer

**Reply by bennothommo** [reply]
@SaeedAhmed99 easiest way is to look at our [Awesome Winter](https://github.com/wintercms/awesome-wintercms) repo, which lists all the plugins we're aware of currently. Each plugin should either link to a GitHub repo or Packagist. If it's a GitHub repo, look for the `composer.json` file and in that should be a `name` value like `author/plugin`. If it's Packagist, the name should be at the very top.

Then, go to your Winter project and type in the following:

```
composer require author/plugin
```

For example, to install the Winter Blog plugin, it would be:

```
composer require winter/wn-blog-plugin
```

There should be a lot of documentation on how to use Composer out there, so feel free to Google it if you get stuck.
